src/Native/Common/pal_config.h.in

```diff
@@ -21,9 +21,12 @@
 #cmakedefine01 HAVE_FDS_BITS
 #cmakedefine01 HAVE_PRIVATE_FDS_BITS
 #cmakedefine01 HAVE_ALLOCA_H
+#cmakedefine01 HAVE_PTHREAD_H
```

Comment:
Makes sense. I was thinking perhaps on Windows it might not be available (if System.Native is supposed to build there).